sdk/storage/storage-blob/samples/javascript/advanced.js

```diff
@@ -1,10 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 /*
  Setup: Enter your storage account name, SAS and a path pointing to local file in main()
 */
 
 const fs = require("fs");
 const { AbortController } = require("@azure/abort-controller");
-const { AnonymousCredential, BlobServiceClient, newPipeline } = require("../.."); // Change to "@azure/storage-blob" in your package
+const { AnonymousCredential, BlobServiceClient, newPipeline } = require("@azure/storage-blob");
+
+const { runSample } = require("./sampleHelpers");
```

Comment:
Not sure if this is addressed already.. this would confuse users?

Reply:
Offline discussion with Will..
- We are going to provide samples through docs.microsoft.com similar to below
 https://docs.microsoft.com/en-ca/samples/browse/?products=azure-storage&languages=javascript%2Cnodejs
  - separate zipped folders for javascript and typescript samples - just like the `Download Zip` link [here](https://docs.microsoft.com/en-ca/samples/azure-samples/storage-blob-upload-from-webapp-node-v10/azure-storage-blob-upload-from-a-nodejs-web-application-using-the-v12-sdk/).
  - In the bullet-list of links in the readmes, `docs.microsoft.com/samples` link will be provided instead of the samples folder link 
  - js samples are located at `samples/javascript` whereas the ts samples are at `samples/typescript/src`
- To improve user-convenience of just copy/pasting one single sample, @willmtemple will be looking to remove `./sampleHelpers` part of the samples

@jeremymeng